scripts/tests/toolkit-e2e.sh

```diff
@@ -23,10 +23,23 @@ echo "🎯 Running Toolkit E2E test"
 echo "🧱 NODE_IMAGE: $NODE_IMAGE"
 echo "🧱 TOOLKIT_IMAGE: $TOOLKIT_IMAGE"
 
+# Ensure Docker network exists
+docker network create toolkit-e2e-net || true
+
+# Start a postgres container for the toolkit sync-cache
+docker run -d --rm \
+    --name postgres-test \
+    --network toolkit-e2e-net \
+    -e POSTGRES_USER=test \
+    -e POSTGRES_PASSWORD=test \
```

Comment:
Rather than hardcoded passwords can we generate a random one:
```
if [ ! -f postgres.password ]; then
    uuidgen | tr -d '-' | head -c 16 > postgres.password
fi
```